samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/prj.conf

```diff
@@ -80,3 +80,41 @@ CONFIG_BT_MESH_LABEL_COUNT=3
 #CONFIG_BT_MESH_SELF_TEST=y
 
 CONFIG_GPIO=y
+
+#### SMP_SVR Related Stuff (START) ####
+
+# Enable mcumgr.
+CONFIG_MCUMGR=y
+
+# Ensure an MCUboot-compatible binary is generated.
```

Comment:
Why is this being set to `n` if an MCUboot compatible binary is desired? That will have the opposite effect.

In general I think if you want to get this in for v1.14 your best hope is to split this all out into a separate config fragment that can get pulled in with CONF_FILE. See https://docs.zephyrproject.org/latest/application/index.html#application-configuration for details.

Even that seems like it might be a bit of a stretch, I am not sure what the maintainers of this app think.